kayenta-atlas/src/main/java/com/netflix/kayenta/atlas/metrics/AtlasMetricsService.java

```diff
@@ -115,6 +117,34 @@ private AtlasNamedAccountCredentials getCredentials(String accountName) {
     AtlasNamedAccountCredentials credentials = getCredentials(accountName);
     BackendDatabase backendDatabase = credentials.getBackendUpdater().getBackendDatabase();
     String uri = backendDatabase.getUriForLocation(URI_SCHEME, atlasCanaryScope.getLocation());
+
```

Comment:
Its a bit of a nit, but the URI resolving logic is starting to get a bit long, maybe it would make sense to refactor / extract it into its own method at some point.

If you were running something like Sonar it might complain about the `Cyclomatic complexity` of this method.

Reply:
Yea, I totally understand.  This is all going to be cleaned up -- soon.  For now, though, I'll merge.